terrain/voxel_block.cpp

```diff
@@ -7,27 +7,32 @@
 // Faster version of Mesh::create_trimesh_shape()
 // See https://github.com/Zylann/godot_voxel/issues/54
 //
-static Ref<ConcavePolygonShape> create_concave_polygon_shape(Array surface_arrays) {
+static Ref<ConcavePolygonShape> create_concave_polygon_shape(Vector<Array*> surfaces) {
```

Comment:
No need for it to be `Array*`, the Godot `Array` type is reference-counted so it can be passed around for very little cost. Besides, Godot's `Vector<T>` also works that way so that's why it is also passed without `*` or `&`.